dataset/filter.py

```diff
@@ -33,18 +34,6 @@
 GOOD_STARTS = GOOD_STARTS_EN | GOOD_STARTS_ZH | GOOD_STARTS_FR | GOOD_STARTS_ES | GOOD_STARTS_PT | GOOD_STARTS_RU | GOOD_STARTS_KO | GOOD_STARTS_JP
 GOOD_ENDS = GOOD_ENDS_KO | GOOD_ENDS_JP
 
-MODEL = "bloomz"
-LANGUAGES = ["python", "java", "javascript"]
-
-if MODEL == "bloomz":
-    LANGUAGES += ["rust", "go", "c++"]
-elif MODEL == "codegeex":
-    # objective-c is the only one missing; Likely partly mixed in with C in the commits data
-    LANGUAGES += [
-        "rust", "go", "c++", "c", "html", "shell", "php", "html+php", "css", "typescript", "sql", "tex", 
-        "objective-c++", "scala", "kotlin", "pascal", "fortran", "r", "cuda", "c#"
-    ]
-
 # 1.0 mean keep all short commit messages
 SHORT_SAMPLING = 1.0
 LONG_SAMPLING = 0.1
```

Comment:
We may remove `SHORT_SAMPLING` and `LONG_SAMPLING` since we do not use them now.

Reply:
Also for `FULL_RANGE_FRAC` because we always use full range content now (and yes it is effective!)